CONTRIBUTING.md

```diff
@@ -122,6 +122,15 @@ To watch the logs of this IntelliJ instance, run `tail -f build/idea-sandbox/sys
 See [Robot plugin](https://github.com/JetBrains/intellij-ui-test-robot)
 and [a preso on testing UI of IntelliJ Plugins](https://slides.com/plipski/intellij-plugin-ui-testing) for more details.
 
+### macOS
+It is not possible to run `uiTest` gradle task on macOS systems directly via `./gradlew`.
+You can run tests inside docker container (based on `gitmachete/intellij-plugin-ci:6.2.0` image) or using virtual machine.
+Sample configuration for launching docker container is shown below.
+```shell
+docker run -v [path to git-machete-intellij-plugin root directory e.g. "$PWD"]:/root/git-machete-intellij-plugin -v ~/.gradle:/root/.gradle -w /root/git-machete-intellij-plugin gitmachete/intellij-plugin-ci:6.2.0 ./gradlew -Pagainst=2021.2 -Pheadless uiTest
```

Comment:
Also! worth adding
`-v ~/.pluginVerifier:/root/.pluginVerifier -v /tmp/ide-probe/cache:/tmp/ide-probe/cache`

so that ide-probe can re-use the IntelliJs downloaded locally

Reply:
Plus add `--rm` so that container is removed after completion... otherwise local Gradle won't be able to e.g.:
```
Starting a Gradle Daemon, 1 busy and 2 incompatible and 7 stopped Daemons could not be reused, use --status for details

FAILURE: Build failed with an exception.

* What went wrong:
java.io.UncheckedIOException: java.io.IOException: Unable to delete directory '/home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources'
  Failed to delete some children. This might happen because a process has files open or has its working directory set in the target directory.
  - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle/accessors/dm/LibrariesForLibs.java
  - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle/accessors/dm
  - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle/accessors
  - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle
  - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org
> java.io.IOException: Unable to delete directory '/home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources'
    Failed to delete some children. This might happen because a process has files open or has its working directory set in the target directory.
    - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle/accessors/dm/LibrariesForLibs.java
    - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle/accessors/dm
    - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle/accessors
    - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org/gradle
    - /home/plipski/git-machete-intellij-plugin/.gradle/7.4.2/dependencies-accessors/2e63973890a031160de15c2136640c23e22ba214/sources/org

```

Reply:
Oh! also... there's a problem that any files are written to any mounted folder... will be owned by the **root**, not the local user on the host machine ☹️ probably setting up uid/gid to run explictly will be needed... if easily doable :/

Reply:
or maybe I can run container as non root ?

Reply:
yes, exactly, pls try

Reply:
> And for what purpose ~/.pluginVerifier should be mounted ?

See ideprobe.conf -> `probe.resolvers.intellij.repositories`... we re-use the IDE downloaded by plugin verifier